src/docs/development/platform-integration/web.md

```diff
@@ -68,19 +67,13 @@ Safari (mobile & desktop)
 Edge (mobile & desktop)
 Firefox (desktop)
 
-During development, Chrome is the default browser for running your app. 
+During development, Chrome and Edge(only on Windows devices) are supported as the default browsers for debugging your app. 
```

Comment:
Suggest: "Chrome (on macOS, Windows and Linux) and Edge (on Windows) are supported as..." 

to avoid any confusion as to whether the parentheses apply to one or both browsers.